### ISSUE TITLE
Show pattern formats in intellisense for Guid.ToString

### ISSUE THREAD

**Issue by medoni** [dotnet/roslyn#46867]
**Brief description:**
Show intellisense suggestions for Guid.ToString(). 

**Languages applicable:**
C# + VB

**Code example that the analyzer should report:**
```csharp
Console.WriteLine(Gui.NewGuid().ToString("N"));
```

**Additional information:**
Same like https://github.com/dotnet/roslyn/issues/17011

**Comment by CyrusNajmabadi** [comment]
@jnm2 you were working on this right?

**Comment by CyrusNajmabadi** [comment]
Closing out due to lack of feedback.
